File: training-management-service/src/main/java/com/otbs/training/dto/TrainingResponseDTO.java
package com.otbs.training.dto;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.List;

public record TrainingResponseDTO(
        Long id,
        String title,
        String description,
        String department,
        LocalDate startDate,
        LocalDate endDate,
        String createdBy,
        List<InvitationResponseDTO> invitations,
        LocalDateTime createdAt
) {
}

File: app/src/main/java/com/example/onetechbs/db/TrainingResponseDTO.kt
package com.example.onetechbs.db

data class TrainingResponseDTO(
    val id: Long,
    val title: String,
    val description: String,
    val department: String,
    val startDate: String,  // use String or LocalDate if using a converter
    val endDate: String,
    val createdBy: String,
    val invitations: List<InvitationResponseDTO>,
    val createdAt: String
)
